src/postgres/tree/PostgresFunctionTreeItem.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureTreeItem, ISubscriptionContext, TreeItemIconPath } from "vscode-azureextensionui";
+import { getThemeAgnosticIconPath } from "../../constants";
+import { PostgresFunctionsTreeItem } from "./PostgresFunctionsTreeItem";
+
+export class PostgresFunctionTreeItem extends AzureTreeItem<ISubscriptionContext> {
+    public static contextValue: string = 'postgresFunction';
+    public readonly contextValue: string = PostgresFunctionTreeItem.contextValue;
+    public readonly schema: string;
+    public readonly name: string;
+    public readonly id: string;
+    public readonly isDuplicate: boolean;
+    public definition: string;
+
+    constructor(parent: PostgresFunctionsTreeItem, schema: string, name: string, oid: number, definition: string, isDuplicate: boolean) {
```

Comment:
I would make an interface for this type: `{ schema: string, name: string, oid: number, definition: string }` and pass the row object into the constructor instead of passing each property individually. I try to avoid having so many parameters with the same type (`string`) because it can be easy to get them out of order and cause a bug